r/R/trackplots.R

```diff
@@ -450,6 +478,126 @@ trackplot_gene <- function(transcripts, region, exon_size = 2.5, gene_size = 0.5
   wrap_trackplot(plot, height=ggplot2::unit(1, "null"))
 }
 
+#' Plot peak tracks
```

Comment:
Given our general name, we should maybe say "Plot range-based annotation track (e.g. peaks)"

Reply:
Agreed